src/labels_by_user_input.py

```diff
@@ -36,6 +44,24 @@ def labels_by_user_input(event_data, pull):
             context=BLOCK_MERGE_VERIFY_CONTEXT,
         )
 
+    (
+        semver_status_state,
+        semver_status_description,
+        semver_labels_to_add,
+        semver_labels_to_remove,
+    ) = get_semver_label_data(pull=pull, body=body)
+    semver_modified = add_remove_labels(
+        pull=pull,
+        labels_to_add=semver_labels_to_add,
+        labels_to_remove=semver_labels_to_remove,
+    )
```

Comment:
```
    semver_modified = add_remove_labels(
        pull=pull,
        labels_to_add=semver_labels_to_add,
        labels_to_remove=semver_labels_to_remove,
    )
```
This should be removed, and you need to check here if add/remove labels and call the proper functions. (add_labels/remove_labels)

Reply:
Sorry, but I completely disagree. Calling add/remove labels here directly adds additional complexity to this block that is not necessary. The current implementation retains add/remove process and create_status calls within this function, while keeping it simple and straight forward. Please reconsider this and other related comments.